ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -43,17 +43,18 @@ tests:
         {
           "failure_rules":
             [
-              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster"},
-              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators"},
-              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other"},
+              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
```

Comment:
Sorry i dont recall but what was the priority for?  Is it for jira tickets

Reply:
priority is for the [group feature in firewatch](https://github.com/CSPI-QE/firewatch/blob/main/docs/configuration_guide.md#group)

basically we group together certain steps so that we can control how many tickets get created by firewatch when one failed step leads to other cascading failures. For example if in the "cluster" group the step ipi-* fails then there is a good chance the must-gather step will fail since there was something wrong with cluster deployment. This group feature allows us to not create a new issue for a failed must-gather step in this example but just create a ticket for the real issue which would be cluster provisioning failure

Reply:
You've also made me notice that the firewatch rules here a bit to vague and we could benefit from making them a bit more specific. I'll update this here.

Reply:
Got it. Thanks for the explanation. You mean the name of rule being vague?